cfgov/v1/handlers/blocks/feedback.py

```diff
@@ -59,22 +59,37 @@ def get_response(self, form):
 
     def success(self):
         if self.request.is_ajax():
-            return JsonResponse({'result': 'pass'})
+            if get_feedback_type(self.block_value) == 'suggestion':
+                return JsonResponse(
+                    {'result': 'pass',
+                     'heading': 'Thank you!',
+                     'message': "Be sure to also sign up for our email list "
```

Comment:
Would it be possible to have this success message be configurable on the block, instead of hardcoding it here? This makes the feedback block OAH-specific.